### PR TITLE
Create chroot install guide, and bootloader page.

### PR DESCRIPTION
**Pull request by pltrz**
The bootloader page is info I gathered from the Arch and Void wikis, I'm not sure if I've put the info down correctly but as it was suggested that bootloader installation/configuration gets it's own page, it's a start.

### CODE REVIEW

File: src/SUMMARY.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
     - [Partitioning Notes](./installation/liveimages/partitions.md)
     - [Installation Guide](./installation/liveimages/guide.md)
   - [Guides](./installation/guides/index.md)
+    - [Installation Guide (chroot)](./installation/guides/chroot-install.md)
```
Comment (CameronNemo) [inline src/SUMMARY.md]: You haven't created the index for the boot loader section.

Comment (pltrz) [reply src/SUMMARY.md]: Updated

**Comment by CameronNemo** [comment]
Can you put the partitioning changes in a separate PR, and squash the small edits commit into one of the other commits?

**Comment by pltrz** [comment]
> Can you put the partitioning changes in a separate PR, and squash the small edits commit into one of the other commits?

Yes, I'll get to that sometime tomorrow

File: src/installation/guides/chroot-guide.md [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+# loadkeys dvorak
+```
+
+### Verify the boot mode
+
+Many new motherboards will support booting in UEFI mode, but some will not,
+instead booting in legacy BIOS mode. You will need to know in which mode your
+system will boot when it is time to partition the disks.
+
+If you have boot via UEFI mode, the directory `/sys/firmware/efi` should exist
```
Comment (the-maldridge) [inline src/installation/guides/chroot-guide.md]: When in UEFI mode, the directory /sys/firmware/efi should be present.  If you believe you should be in EFI mode and this directory is not present, verify that the efivars filesystem is mounted.

**Comment by pltrz** [comment]
@the-maldridge Understood. I'll restart and try again. Could you elaborate on some things you think I should be doing differently?

You mention:
> Its full of too many magic commands and no substance as to how to install the system and why each step is happening.

I'm having trouble understanding what you really mean and by extension how to fix it.

I didn't realize it was a bad guide, but I'm committed to getting it right. Let me know of any advice for the redo.

**Comment by the-maldridge** [comment]
I think its a fine guide, but I don't think it is suitable for this repo.  What I mean by magic commands is that in several places it says to run some commands and doesn't really explain why you should run them, or what the end goal is.

I would much prefer a guide that walks through the phases of an install, with examples shown for each, but written in such a way that it can't be blindly copy/pasted, since that would encourage installation by magic, rather than by knowledge.  For reference, I am using sense 1 from here: http://www.catb.org/jargon/html/M/magic.html.  Nothing in the install should be too complicated to explain, and any guide which tries to walk through an install should be able to do so more abstractly.

A concrete example: rather than saying to use a specific partition layout, I would expect a well written guide to recommend characteristics of the disk such as a separate boot, the size of this boot partition, and a short discussion as to whether or not swap space is really necessary, and how much.

If you want to write a guide that's copy/paste/void then that can go in the guides/ subsection perhaps under the title of "Chroot install in N steps" where N is the number of steps.

**Comment by pltrz** [comment]
Thanks @the-maldridge.

It's clearer now, I should be really diving into each section so that the reader is in the driver's seat and making informed decisions based on the guide, and the guide itself is really more general information into the options available at each step, so that the user can decide for himself, instead of just giving commands with brief context on what's happening (*magic*). In a sense it should really be more of a handbook. My understanding is superficial but I'm getting what you mean a bit more.

Thanks for following up; I see now that the guide should be something different than I originally thought. I expect you have a much clearer vision for how this guide should be than I do, and so for that reason I also expect there might be a few revisions until it's done right. I think this puts me more on the right track, though. I'll be starting the guide with a shift in objective.

**Comment by the-maldridge** [comment]
Yeah, that's really what I'm intending, a repository of information that the reader can use to make decisions for themselves.  The guides/ section though is designed to contain copy/paste instructions for specific platforms if that's needed (think installing Void in places that don't use the installer, for example).

**Pull request closed without merging**